<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>leftPart</class>
 <widget class="QWidget" name="leftPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="openDir">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Open
Dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Next
Image</string>
   </property>
   <property name="shortcut">
    <string>D, Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Prev
Image</string>
   </property>
   <property name="shortcut">
    <string>A, Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Create
Label</string>
   </property>
   <property name="shortcut">
    <string>A, Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete
File</string>
   </property>
   <property name="shortcut">
    <string>A, Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>A, Left</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
